common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -90,6 +101,15 @@ class AccountInfo extends React.Component<Props, State> {
           </div>
           <div className="AccountInfo-address-wrapper">
             <div className="AccountInfo-address-addr">{address}</div>
+            <CopyToClipboard onCopy={this.onCopy} text={address}>
+              <div
+                className={`AccountInfo-copy-icon${this.state.copied ? '-copied' : ''}`}
+                title="Copy To Clipboard"
+              >
+                <i className="fa fa-copy" />
+                <p>{this.state.copied ? 'copied!' : null}</p>
```

Comment:
Right now this text appears on a second line, I think it'd look a little nicer if it was besides the button. Can you make this a span, and add a margin between them?

Reply:
Also, because some users may not understand the icon, it'd be nice if when it wasn't copied, there was some text that said `copy address`.